index.html

```diff
@@ -1 +1,980 @@
+                </div>
+            </div>
+            <div class="articleImage">
+                <div>
+                    <img src="assets/add3.jpg" alt="add3 image">
+                    <div>
+                        <p>NASA’s Asteroid Mission Packs Away Its Cargo. Next Stop: Earth.</p>
+                    </div>
+                    <span>Nov. 2</span>
+                </div>
+            </div>
+        </section>
+    </article>
+    <div class="bottom-side-bar">
+        <h4>Most Popular</h4>
+        <div class="side-bar-margin">
+            <a href='#'>
+                <p> What’s Left in the Seven States That Will Decide the Race</p>
+            </a>
+            <a href='#'>
+                <p>Can Biden Still Win? Can Trump Still Win? Yes. Here Are the Remaining Paths.</p>
+            </a>
+            <a href='#'>
+                <p>The Remaining Vote in Pennsylvania Appears to Be Overwhelmingly for Biden</p>
+            </a>
+            <a href='#'>
+                <p>Lauren Boebert, Hard-Right Gun Activist, Wins in Colorado House District</p>
+            </a>
+            <a href='#'>
+                <p>Fox News Made a Big Call in Arizona, Buoying Biden and Angering Trump</p>
+            </a>
+            <a href='#'>
+                <p>$1.25 Million Homes in Maine, Texas and New York</p>
+            </a>
+            <a href='#'>
+                <p>In Voting Cases, Chief Justice Roberts Is Alone but in Control</p>
+            </a>
+            <a href='#'>
+                <p>Dementia ‘Took Its Toll’ on Sean Connery, Wife Says</p>
+            </a>
+            <a href='#'>
+                <p>What We Know About the Vote in Wisconsin, Michigan and Pennsylvania</p>
+            </a>
+            <a href='#'>
+                <p>When He Bought the Tiny House, He Was Single. That Didn’t Last.</p>
+            </a>
+        </div>
+    </div>
+    <br>
+    <br>
+    <br>
+    <div class="footer-border"></div>
+    <section class="bottom-nav">
+        <div class="bottom-head">
+            <a href='#'><img src="./assets/download.png" alt="Main-logo"></a>
+            <a class="bottom-head-span"><span>Go to Home Page</span></a>
+        </div>
+        <br>
+        <section class="bottom-header-nav">
+            <div>
+                <b>NEWS</b>
+                <a href='#'>
+                    Home Page
+                </a>
+                <a href='#'>
+                    World
+                </a>
+                <a href='#'>
+                    U.S.
+                </a>
+                <a href='#'>
+                    Politics
+                </a>
+                <a href='#'>
+                    Election 2020
+                </a>
+                <a href='#'>
+                    New York
+                </a>
+                <a href='#'>
+                    Business
+                </a>
+                <a href='#'>
+                    Tech
+                </a>
```

Comment:
- [ ] This breaks readability, only use new-lines for lines of code that are larger than 80 chars. [OPTIONAL]

This would improve readablity of the file if are set up like this:
```
...
<a href='#'>Tech</a>
<a href='#'>Business</a>
<a href='#'>New York</a>
...
```

Reply:
Thank you and made the changes as suggested.